flink/src/main/java/org/apache/sedona/flink/Catalog.java

```diff
@@ -32,6 +32,9 @@ public static UserDefinedFunction[] getFuncs() {
                 new Constructors.ST_GeomFromGeoJSON(),
                 new Constructors.ST_GeomFromGeoHash(),
                 new Functions.ST_Buffer(),
+                new Functions.ST_MakeValid(),
+                new Functions.ST_Intersection(),
+                new Functions.ST_Difference(),
```

Comment:
Can you just make a separate PR for these and add them to the new common module created in https://github.com/apache/incubator-sedona/pull/647? Even though they're pretty simple it will help not to maintain both definitions in flink and spark

Reply:
Yeah, just saw that #647 got merged earlier this morning, so i have to merge it in my branch anyways. I'll clean it up and make a seperate PR for them.

Reply:
I pushed a rebased version with #647  only containing the ST_Collect function. 
Apparently #647 broke the build process for me, see: https://github.com/apache/incubator-sedona/pull/647#issuecomment-1216570079.